app/views/job_invocations/_tab_overview.html.erb

```diff
@@ -9,7 +9,19 @@
   <% template_invocations.each do |template_invocation| %>
     <%= minicard('user', template_invocation.effective_user || Setting[:remote_execution_effective_user],
                  template_invocation.template.name + ' ' + _('effective user')) %>
-    <%= minicard('cluster', job_invocation.total_hosts_count, _('Total hosts')) %>
+    <div class="row">
+      <% missing = job_invocation.targeting.resolved? ? job_invocation.total_hosts_count - job_invocation.targeting.hosts.count : 0 %>
```

Comment:
should this be a `job_invocation` method instead? we should probably add the same information to the API

Reply:
Done, api now shows this, hammer pr will follow shortly